### PR TITLE
T667

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: src/butter-main.js [hunk 1]
```diff
@@ -78,7 +82,7 @@
       this.getHTML = function(){
         var media = [];
         for( var i=0; i<_media.length; ++i ){
-          media.push( _media[ i ].popcornString );
+          media.push( _media[ i ].generatePopcornString() );
```
Comment (brianchirls) [inline src/butter-main.js:85]: I like this. Much less confusing.

File: test/core.js [hunk 2]
```diff
@@ -638,4 +642,37 @@
...
     });
   });
+
+  module( "Popcorn scripts and callbacks");
+  asyncTest( "Existence and execution", function(){
+    expect( 6 );
+
+    createButter( function( butter ){
+
+      var theZONE = "";
```
Comment (dseif) [inline test/core.js:652]: http://www.youtube.com/watch?feature=player_detailpage&v=MvpcSrrFphQ#t=17s

Comment (secretrobotron) [inline test/core.js:652]: Noooo no no no no http://www.youtube.com/watch?v=0O_IHOwgW0w

**Comment by jbuck** [comment]
Obsoleted by pull request #202

**Pull request closed without merging**